spec/lib/core/entity/article_spec.rb

```diff
@@ -124,6 +124,64 @@ module Core
       end
     end
 
+    describe '#latest_blog_post_links' do
+      context 'provide data' do
+        let(:related_content) do
+          {
+            'latest_blog_post_links' => [
+              {
+                'title' => 'Latest post.',
+                'path' => '/latest-post',
+                'date' => '2015-01-02'
+              },
+              {
+                'title' => 'Oldest post.',
+                'path' => '/oldest-post',
+                'date' => '2015-01-01'
+              }
+            ]
+          }
+        end
+
+        it 'has 2 article links' do
+          expect(subject.latest_blog_post_links.length).to eq(2)
+          expect(subject.latest_blog_post_links.first).to be_an_instance_of(ArticleLink)
```

Comment:
I'd probably split these into two different specs .... Somthing like 'has 2 articles' and 'all links are ArticleLink' :)